app/views/layouts/flood_risk_engine.html.erb

```diff
@@ -28,4 +28,4 @@
   <%= content_tag :div, yield %>
 <% end -%>
 
-<%= render template: 'layouts/govuk_admin_template' %>
+<%#= render template: 'layouts/govuk_admin_template' %>
```

Comment:
Should we delete this entirely?

Reply:
Yes, it's to render the old template, which was installed with the 'govuk_admin_template' gem, but is no longer there.

Reply:
On closer inspection - this layout renders the engine / journey pages. We have a ticket to integrate this, so I think we should leave the file as is, for now.